sections/image-banner.liquid

```diff
@@ -24,12 +24,29 @@
 {%- endif -%}
 
 {%- style -%}
+  #Banner-{{ section.id }} .banner__box {
+    text-align: {{ section.settings.mobile_text_alignment }};
+  }
+
+  @media screen and (min-width: 750px) {
+    #Banner-{{ section.id }} .banner__box {
+      text-align: {{ section.settings.text_alignment }}
+    }
+  }
```

Comment:
@ludoboludo I have tested it and both mobile and desktop seem to work 🤔 
Quick video: https://screenshot.click/22-31-t8yet-022qj.mp4

Do you mind re-testing and if you still see this issue telling me the exact settings you have?

Reply:
Good catch, Ludo! Fixed here: https://github.com/Shopify/dawn/pull/900/commits/a1aef4bca954c86939f6e150394e30ebb0255110#diff-8a48cf1c4c77bd24f3d47ceac2c19b0149a25b33c5e4f3d3155d249448ffdcfaR299-R307

This was actually because of the alignment on those lines being updated, but not according to the mobile settings

Reply:
This is also happening on the email banner PR.